### ISSUE TITLE
No Drop - error

### ISSUE THREAD

**Issue by UlyanaLeyana** [O-n-y/oxy.mods#42]
Got this some minutes ago. Have latest version of the mod v 1.0.8.67. I think no reason to put save file here because previous save was 2 cycles before getting this error. 

```
NullReferenceException

UnityEngine.GameObject.GetComponent[T] () (at <b8388ffefd6046f8a7b4080349da72eb>:0)
Ony.OxygenNotIncluded.NoDrop.FetchChore_FindFetchTarget.Prefix (FetchChore __instance, Pickupable& __result, ChoreConsumerState consumer_state) (at <e74030c102fb4a48a16118134c60d3de>:0)
(wrapper dynamic-method) FetchChore.FindFetchTarget_Patch2(object,ChoreConsumerState)
FetchChore+<>c.<.cctor>b__40_0 (Chore+Precondition+Context& context, System.Object data) (at <f834fb9792e545c0aa06cca79073875a>:0)
Chore+Precondition+Context.RunPreconditions () (at <f834fb9792e545c0aa06cca79073875a>:0)
Chore.CollectChores (ChoreConsumerState consumer_state, System.Collections.Generic.List`1[T] succeeded_contexts, System.Collections.Generic.List`1[T] failed_contexts, System.Boolean is_attempting_override) (at <f834fb9792e545c0aa06cca79073875a>:0)
FetchChore.CollectChoresFromGlobalChoreProvider (ChoreConsumerState consumer_state, System.Collections.Generic.List`1[T] succeeded_contexts, System.Collections.Generic.List`1[T] failed_contexts, System.Boolean is_attempting_override) (at <f834fb9792e545c0aa06cca79073875a>:0)
GlobalChoreProvider.CollectChores (ChoreConsumerState consumer_state, System.Collections.Generic.List`1[T] succeeded, System.Collections.Generic.List`1[T] failed_contexts) (at <f834fb9792e545c0aa06cca79073875a>:0)
ChoreConsumer.FindNextChore (Chore+Precondition+Context& out_context) (at <f834fb9792e545c0aa06cca79073875a>:0)
Brain.FindBetterChore (Chore+Precondition+Context& context) (at <f834fb9792e545c0aa06cca79073875a>:0)
Brain.UpdateChores () (at <f834fb9792e545c0aa06cca79073875a>:0)
Brain.UpdateBrain () (at <f834fb9792e545c0aa06cca79073875a>:0)
MinionBrain.UpdateBrain () (at <f834fb9792e545c0aa06cca79073875a>:0)
BrainScheduler+BrainGroup.RenderEveryTick (System.Single dt, System.Boolean isAsyncPathProbeEnabled) (at <f834fb9792e545c0aa06cca79073875a>:0)
BrainScheduler.RenderEveryTick (System.Single dt) (at <f834fb9792e545c0aa06cca79073875a>:0)
SimAndRenderScheduler+RenderEveryTickUpdater.Update (IRenderEveryTick updater, System.Single dt) (at <4c2b2fae6abe493697de2dfee51c79b6>:0)
UpdateBucketWithUpdater`1[DataType].Update (System.Single dt) (at <4c2b2fae6abe493697de2dfee51c79b6>:0)
StateMachineUpdater+BucketGroup.AdvanceOneSubTick (System.Single dt) (at <4c2b2fae6abe493697de2dfee51c79b6>:0)
StateMachineUpdater.RenderEveryTick (System.Single dt) (at <4c2b2fae6abe493697de2dfee51c79b6>:0)
Game.LateUpdate () (at <f834fb9792e545c0aa06cca79073875a>:0)
```

**Comment by O-n-y** [comment]
Please provide a save file where this could be reproduced, I would not able to.

**Comment by O-n-y** [comment]
Please use last released version which is  1.1.1.16.
Your is outdated
